src/main/java/com/epam/processor/DataProcessor.java

```diff
@@ -51,7 +70,20 @@ public RoadAccident getAccidentByIndex7(String index){
      * @return
      */
     public Map<String, Long> getCountByRoadSurfaceCondition7(){
-        return null;
+        return getCountBy(RoadAccident::getRoadSurfaceConditions);
```

Comment:
Technically it's not J7 :) 
I think this syntax is from J8...

But yea - it's fine